### DISCUSSION TITLE
Lesson 13:  Timestamp 14:26 saving and updating UI without having to run yarn start

### DISCUSSION THREAD

**Discussion by ChristianG77** [smartcontractkit/full-blockchain-solidity-course-py#328] [Q&A] [answered]
Hi

Do you always have to run 'yarn start' to update the browser UI after making changes to the code?

The video seems to suggest you just need to save and the browser will automatically update.  It takes a decent amount of time to run 'yarn start'.  Maybe my laptop is slow but it would be helpful if there was a work around.

Thanks.

**Comment by cromewar** [comment, marked as answer]
Hello @ChristianG77... well this is tricky because you actually need to use `yarn start` because it starts a local server and compiles your code,  it is by itself a lightweight solution rather than using another local server and connect manually. So in this case I'll suggest you to close any other program and close other tools you are not using at the moment, also open a new terminal and run yarn start just once so it'll just update the changes rather start over and over again.

**Reply by ChristianG77** [reply]
Thanks!
